apps/docs/src/guide/transactions/transaction-policies.md

```diff
@@ -0,0 +1,20 @@
+# Transaction Policies
+
+Transaction policies are rules that can govern how a transaction is processed, based off the transaction parameters that are passed to a transaction request. The available policies are as follows:
```

Comment:
Also, what's the difference between setting transaction parameters and setting transaction policies? Both seem to have the same set of fields.